crawler.py

```diff
@@ -1,32 +1,43 @@
+import requests
+
 from bs4 import BeautifulSoup
 
-import requests
 
-def parse():
+def crawl():
+    headers = {
+        'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 '
+                      '(KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36'
+    }
+
+    games = []
 
-	HEADERS = {
-		'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36'
```

Comment:
In your code tabs were used for formatting.

I Python code intendation MUST be done with spaces (4 spaces for one level of intendation).

In case you use any decent tool designed for work with Python code, this setting is applied by default.